.werft/jobs/build/build-and-publish.ts

```diff
@@ -79,9 +79,6 @@ export async function buildAndPublish(werft: Werft, jobConfig: JobConfig) {
                 );
             }
 
-            werft.phase("publish", "publishing Helm chart...");
-            publishHelmChart(werft, "gcr.io/gitpod-io/self-hosted", version);
-
             werft.phase("publish", `preparing GitHub release files...`);
```

Comment:
We also haven't been using this path of the job through the `publish-release` arg. I thought of removing it, but wasn't sure if we get back to this automated way of doing this in the future. 🤔